source/2019-04-16-ember-3-9-released.md

```diff
@@ -0,0 +1,94 @@
+---
+title: Ember 3.10 Released
+author: Kenneth Larsen
+tags: Releases, 2019, 3, 3.10
+responsive: true
+---
+
+Today the Ember project is releasing version 3.10 of Ember.js, Ember Data, and Ember CLI. This release kicks off the 3.11 beta cycle for all sub-projects. We encourage our community (especially addon authors) to help test these beta builds and report any bugs before they are published as a final release in six weeks' time. The [ember-try](https://github.com/ember-cli/ember-try) addon is a great way to continuously test your projects against the latest Ember releases.
+
+You can read more about our general release process here:
+
+- [Release Dashboard](http://emberjs.com/builds/)
+- [The Ember Release Cycle](http://emberjs.com/blog/2013/09/06/new-ember-release-process.html)
+- [The Ember Project](http://emberjs.com/blog/2015/06/16/ember-project-at-2-0.html)
+- [Ember LTS Releases](http://emberjs.com/blog/2016/02/25/announcing-embers-first-lts.html)
+
+---
+
+## Ember.js
+
+Ember.js is the core framework for building ambitious web applications.
+
+### Changes in Ember.js 3.10
+Ember.js 3.10 is an incremental, backwards compatible release of Ember with bugfixes, performance improvements, and minor deprecations. There is COUNT (#) new feature, COUNT (#) deprecations, and COUNT (#) bugfixes in this version.
+
+#### New Features (2)
```

Comment:
Enabled:

* [emberjs/rfcs#398](https://emberjs.github.io/rfcs/0398-RouteInfo-Metadata.html) "RouteInfo Metadata".
* [emberjs/rfcs#440](https://emberjs.github.io/rfcs/0440-decorator-support.html) "Native Decorator Support".
* [emberjs/rfcs#457](https://emberjs.github.io/rfcs/0457-nested-lookups.html) "Angle Bracket Nested Lookup".
* [emberjs/rfcs#459](https://emberjs.github.io/rfcs/0459-angle-bracket-built-in-components.html) "Angle Bracket Built-ins".